### PR TITLE
Unit test 1

### PR DESCRIPTION
**Pull request by rbhatt1999**
In this branch following changes done:
1-Setup jest.
2- Add test for add & remove function.
3- Add mocks for todolist.
4-Ensure that there are no linter errors.

### CODE REVIEW

File: src/test/crud.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    add('bye');
+    const storage = JSON.parse(localStorage.getItem('todoList'));
+    expect(storage).toHaveLength(3);
+  });
+  test('test for remove funtion', () => {
+    remove(0);
+    const storage = JSON.parse(localStorage.getItem('todoList'));
+    expect(storage).toHaveLength(2);
+  });
+});
```
Comment (muneebulrehman) [inline src/test/crud.test.js]: Great job with testing for the local storage. However, you also need to mock HTML to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/crud.test.js (hunk 1 above)
Comment (rbhatt1999) [reply src/test/crud.test.js]: updated

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @rbhatt1999 @asifaminisonline,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

[OPTIONAL] Kindly add your contribautor handle in the readme file so that it show collaboration and Teamwork. Also replace @handletwitwer by your name it looks more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**